Source/Classes/TextMask.lua

```diff
@@ -105,10 +109,36 @@ function TextMaskClass:SetMaskType(name)
 	end
 	self._MaskType = mask
 	self.Frame.Text = mask:Process(self.Frame.Text):sub(1, self._MaxLength)
+	
+	-- Add increment/decrement buttons for numeric inputs
+	if name == "Number" then
+		-- Initialize text to 0, consumer can override
+		self.Frame.Text = 0
+
+		self.incrementButton = INCREMENT_BUTTON:Clone()
+		self.incrementButton.Position = UDim2.new(0, self.Frame.AbsoluteSize.X - self.Frame.AbsoluteSize.Y / 2, 0, 0)
```

Comment:
Problems could arrise from using absolute size in this context. If the user decided to resize `self.Frame` then the scaling will not carry over.

The same effect can be achieved w/ scale and anchor points.

```suggestion
self.IncrementButton.AnchorPoint = Vector2.new(1, 0)
self.IncrementButton.Position = UDim2.new(1, 0, 0, 0)
```

In addition for sake of matching the rest of the code base the `i` in `increment` should be capitalized.